src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -7,6 +7,10 @@ public interface CinemaHallService {
     CinemaHall add(CinemaHall cinemaHall);
 
     CinemaHall get(Long id);
-    
+
     List<CinemaHall> getAll();
+
+    Boolean update(CinemaHall cinemaHall);
```

Comment:
The return type of the `update` method should be `boolean` instead of `Boolean` to avoid unnecessary boxing.